src/routes/student/Inventory.tsx

```diff
@@ -88,120 +90,140 @@ export default function Inventory({ player, classroom }: { player: Player; class
-					<Tab label='Shirts' {...a11yProps(3)} />
-					<Tab label='Pants' {...a11yProps(4)} />
-					<Tab label='Shoes' {...a11yProps(5)} />
-				</Tabs>
-			</Box>
-
-			<TabPanel value={value} index={0}>
-				<Grid container spacing={{ xs: 2, md: 3 }} columns={{ xs: 4, sm: 8, md: 12 }}>
-					{inventoryObjects.map((item, index) => (
-						<Grid item xs={12} sm={6} md={4} key={index}>
-							{/* <InventoryItemCard item={item} player={player} classroom={classroom} /> */}
-							<ItemCard
-								item={item}
-								player={player}
-								classroom={classroom}
-								itemPrice=''
-								type='inventory'
-							/>
-						</Grid>
-					))}
-				</Grid>
-			</TabPanel>
-
-			<TabPanel value={value} index={1}>
-				<Grid container spacing={{ xs: 2, md: 3 }} columns={{ xs: 4, sm: 8, md: 12 }}>
-					{getBodyItems().map((item, index) => (
-						<Grid item xs={2} sm={3} md={3} key={index}>
-							{/* <InventoryItemCard item={item} player={player} classroom={classroom} /> */}
-							<ItemCard
-								item={item}
-								player={player}
-								classroom={classroom}
-								itemPrice=''
-								type='inventory'
-							/>
-						</Grid>
-					))}
+			<Grid sx={{ display: 'flex', flexDirection: 'column' }} container>
+				<Grid item xs={12}>
+					<Typography variant='h2'>Inventory</Typography>
+					<h5>Equip your avatar with items from your inventory!</h5>
 				</Grid>
-			</TabPanel>
-
-			<TabPanel value={value} index={2}>
-				{inventoryObjects
-					.filter((item) => item.type === 'hair')
-					.map((item, index) => (
-						<Grid item xs={12} sm={6} md={4} key={index}>
-							{/* <InventoryItemCard item={item} player={player} classroom={classroom} /> */}
-							<ItemCard
-								item={item}
-								player={player}
-								classroom={classroom}
-								itemPrice=''
-								type='inventory'
-							/>
-						</Grid>
-					))}
-			</TabPanel>
-
-			<TabPanel value={value} index={3}>
-				{inventoryObjects
-					.filter((item) => item.type === 'shirt')
-					.map((item, index) => (
-						<Grid item xs={12} sm={6} md={4} key={index}>
-							{/* <InventoryItemCard item={item} player={player} classroom={classroom} /> */}
-							<ItemCard
-								item={item}
-								player={player}
-								classroom={classroom}
-								itemPrice=''
-								type='inventory'
-							/>
+				<Box sx={{ borderBottom: 1, borderColor: 'divider' }}>
+					<Tabs value={value} onChange={handleChange} aria-label='basic tabs example'>
+						<Tab label='All' {...a11yProps(0)} />
+						<Tab label='Bodies' {...a11yProps(1)} />
+						<Tab label='Hair' {...a11yProps(2)} />
+						<Tab label='Shirts' {...a11yProps(3)} />
+						<Tab label='Pants' {...a11yProps(4)} />
+						<Tab label='Shoes' {...a11yProps(5)} />
+					</Tabs>
+				</Box>
+				<div style={divStyle}>
+					<TabPanel value={value} index={0}>
+						<Grid container spacing={{ xs: 2, md: 3 }} columns={{ xs: 4, sm: 8, md: 12 }}>
+							{inventoryObjects.map((item, index) => (
```

Comment:
If you have time, make it so that if there are no items to display, display some text saying 'No items' or something